File: Python Word examples/sections/updatePageSetup.py
import spirecloudword
from spirecloudword import LineNumber, PageSetup, Margin
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.sections_api.SectionsApi(configuration)

file_name = "Template.docx"
section_index = 0
linenumber = LineNumber(start_value=2, step=2)
linenumber._restart_mode = "RestartSection"
linenumber._distance_from_text = 0
page_setup = PageSetup()
page_setup._page_width = 800
page_setup._page_height = 500
page_setup._page_margin = Margin(50, 50, 70, 50)
page_setup._orientation = 'Portrait'
page_setup._linenumber = linenumber
extra = {'folder': 'input', 'dest_file_path': 'output/UpdatePageSetup.docx'}
api.update_page_setup(file_name, section_index, page_setup, **extra)

File: JavaScript Word Examples/SectionsApiDemo/updatePageSetup.js
(function (){
    var appId = "your id";
    var appKey = "your key";  
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.SectionsApi(apiClient);

    var fileName = "PageSetup.docx";
    var sectionIndex = 0;
    var pageSetup = new Spirecloudword.PageSetup();
    pageSetup.pageWidth = 400.0;
    pageSetup.pageHeight = 500.0;
    pageSetup.pageMargin = new Spirecloudword.Margin();
    pageSetup.pageMargin.top = 20.0;
    pageSetup.pageMargin.bottom = 20.0;
    pageSetup.pageMargin.left = 20.0;
    pageSetup.pageMargin.right = 20.0;
    pageSetup.linenumber = new Spirecloudword.LineNumber();
    pageSetup.linenumber.startValue = 1;
    pageSetup.linenumber.step = 2;
    pageSetup.linenumber.restartMode = "Continuous";
    pageSetup.linenumber.distanceFromText = 20;
    pageSetup.orientation = "Landscape";
    var destFilePath = 'output/UpdatePageSetup.docx';
    var opts = { folder: 'input' };
    instance.updatePageSetup(fileName, sectionIndex, pageSetup, destFilePath, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();